### PR TITLE
Hotfix/crash due to parsing

### PR DESCRIPTION
**Pull request by SungjunApp**
This fixes app crashing issues due to network data parsing.

### CODE REVIEW

**Comment by SungjunApp** [comment]
The only new commit is **Fix crashing issues**.

I'm sorry for uploading the three commits above except **Fix crashing issues**. These three are already merged into master in the previous PR.

File: app/src/main/java/com/pixlee/pixleeandroidsdk/ui/AnalyticsFragment.java [hunk 1]
```diff
@@ -264,30 +215,31 @@ public void onClick(View view) {
...
             public void onScrollChange(NestedScrollView v, int scrollX, int scrollY, int oldScrollX, int oldScrollY) {
-                if (v_widget != null) {
-                    if (v_widget.getLocalVisibleRect(scrollBounds)) {
+                if (binding.vWidget != null) {
+
+                    if (binding.vWidget.getLocalVisibleRect(scrollBounds)) {
+
                         if (!widgetVisible) {
-                            String widgetVisibleStatus = "widgetVisible " +
+                            String message2 = "visibleWidget " +
```
Comment (jchen123) [inline app/src/main/java/com/pixlee/pixleeandroidsdk/ui/AnalyticsFragment.java]: the message2 is back? and it doesnt seem to even be used, the line below uses message not message2?

Comment (SungjunApp) [reply app/src/main/java/com/pixlee/pixleeandroidsdk/ui/AnalyticsFragment.java]: This was my mistake. I need to use this so I fixed this.

**Comment by SungjunApp** [comment]
If an int variable receives a null, the parsing will be failed. This hotfix will tackle the problem.

**Comment by SungjunApp** [comment]
Since this fix needs to be sent to AP Mall ASAP, I'll merge this without KB's approval. KB Please let me know if there is anything wrong.

**Pull request merged**